### PR TITLE
Numerical Stability of Relative Losses

### PR DESCRIPTION
**Pull request by vduruiss**
Adding an epsilon to the denominators of relative losses to avoid numerical instability when `ynorm` is close to 0

Currently set to `eps=1e-8` since it seems to be a fairly common value in torch/scipy/numpy

### CODE REVIEW

File: neuralop/losses/data_losses.py [hunk 1]
```diff
@@ -64,6 +64,7 @@ def __init__(self, d=1, p=2, measure=1., reduction='sum'):
 
         self.d = d
         self.p = p
+        self.eps = 1e-8
```
Comment (dhpitt) [inline neuralop/losses/data_losses.py:67]: Should this be parameterized?

**Comment by dhpitt** [comment]
Thanks for the PR @vduruiss ! One small question about whether it would be worth parameterizing `self.eps`

**Comment by vduruiss** [comment]
@dhpitt We can certainly parametrize it, either as an argument of the `__init__()` for the loss class, or as an input parameter of the `rel()` function, whichever you think would be best practice

**Comment by JeanKossaifi** [comment]
Thanks! I would keep it in the __init__ so the loss can then be used as a function, without the need for additional arguments, e.g. 
```python
train_loss = Loss(all_args_here)
...
loss = train_loss(ground_truth, prediction)
```

**Comment by dhpitt** [comment]
I agree, that sounds good

**Comment by JeanKossaifi** [comment]
Awesome, merging, thanks you!
